src/main/scala/symsim/examples/concrete/simplemaze/Maze.scala

```diff
@@ -93,9 +94,12 @@ object Maze
       newState = successor (s) (action)
     yield (newState, mazeReward (newState))
 
-  def initialize: Randomized[MazeState] =
-    Randomized.repeat (Randomized.oneOf (instances.allObservableStates*))
-      .filter (s => !isFinal (s))
+  def initialize: Randomized[MazeState] = for
+    x <- Randomized.repeat(Randomized.between(1, 4))
```

Comment:
This might introduce the same initialization problem that I suspect we have in the braking car.

Reply:
OK This is not a problem, as analyzed here: https://github.com/itu-square/symsim/pull/226#issuecomment-1870625502